homeassistant/components/plugwise/sensor.py

```diff
@@ -94,13 +120,15 @@
         native_unit_of_measurement=POWER_WATT,
         device_class=SensorDeviceClass.POWER,
         state_class=SensorStateClass.MEASUREMENT,
+        entity_registry_enabled_default=False,
     ),
     SensorEntityDescription(
         key="electricity_consumed_interval",
         name="Electricity consumed interval",
         native_unit_of_measurement=ENERGY_WATT_HOUR,
         device_class=SensorDeviceClass.ENERGY,
         state_class=SensorStateClass.TOTAL,
+        entity_registry_enabled_default=False,
```

Comment:
Why is it disabled?

Reply:
Also, for Plugs, P1, Stretch, less interesting numbers, per 5 mins or per hour, but the graph updates per 5 min. So completely useless.

Reply:
Disabled by default has nothing to do with something being interesting or not. Therefore this change is incorrect and should be reverted.

Reply:
Reverted.